developer/openshift/gitops/argocd/pipeline-service/tekton-results/minio-bucket-creation.yaml

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
This file shares a lot in common with [s3-configuration-patch.yaml](https://github.com/openshift-pipelines/pipeline-service/blob/main/developer/openshift/gitops/argocd/pipeline-service/tekton-results/s3-configuration-patch.yaml). Should it not be the case that only one of the two files should exist?

Reply:
ok, I moved this part to the same file.